packages/example/src/pages/NonVirtualizedGridPage.tsx

```diff
@@ -0,0 +1,65 @@
+import { DefaultFocus, SpatialNavigationNode } from 'react-tv-space-navigation';
+import { Page } from '../components/Page';
+import '../components/configureRemoteControl';
+import { getPrograms } from '../modules/program/infra/programInfos';
+import { useNavigation } from '@react-navigation/native';
+import { NativeStackNavigationProp } from '@react-navigation/native-stack';
+import { RootStackParamList } from '../../App';
+import styled from '@emotion/native';
+import { scaledPixels } from '../design-system/helpers/scaledPixels';
+import { ProgramNode } from '../modules/program/view/ProgramNode';
+import chunk from 'lodash/chunk';
+import { ProgramInfo } from '../modules/program/domain/programInfo';
+import { theme } from '../design-system/theme/theme';
+
+const renderProgramList = (programsLists: ProgramInfo[]) => (
+  <ProgramList programs={programsLists} />
+);
+
+export const NonVirtualizedGridPage = () => {
+  const programsList = chunk(getPrograms(), 7);
```

Comment:
You could extract this into a local constant in the file to make it more obvious 😁 

```suggestion
  const programsList = chunk(getPrograms(), ROW_SIZE);
```